docs/reference/form_types.rst

```diff
@@ -4,18 +4,17 @@ Form Types
 Admin related form types
 ------------------------
 
-When defining fields in your Admin classes you can use any of the standard
+When defining fields in your admin classes you can use any of the standard
 `Symfony field types`_ and configure them as you would normally. In addition
 there are some special Sonata field types which allow you to work with
 relationships between one entity class and another.
 
 .. _field-types-model:
 
-sonata_type_model
-^^^^^^^^^^^^^^^^^
+``Sonata\AdminBundle\Form\Type\ModelType``
```

Comment:
I think we elected to use just ` ``ModelType`` ` on another PR I made. @sonata-project/contributors , do you remember which repo?

Reply:
so just the class name and not the FQCN, but then there will be a problem with `Sonata\CoreBundle\Form\Type\CollectionType` and `Sonata\AdminBundle\Form\Type\CollectionType`

Reply:
🤔 Hmmm right. We should probably keep it like this then...